consensus/hotstuff_leader.go

```diff
@@ -31,21 +31,28 @@ func (handler *HotstuffLeaderMessageHandler) HandleNewRoundMessage(m *consensusM
 	handler.emitTelemetryEvent(m, msg)
 
```

Comment:
General comment: Please see all the comments in this file to make sure the Step being logged corresponds to the actual function (i.e. what we had before)